Fix precedence of the quanta layer by adding a base layer for all the… (#6539)
plone/volto: docs/source/configuration/settings-reference.md

    [See an explanation of character limits in URLs](https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers/417184#417184).
    Please test this setting properly before enabling in a production site.

cssLayers

>     If you want to use CSS layers in Volto styling, it's possible to define them making sure that they are defined and applied at the very top level of the page (head tag).

Reviewer: ```suggestion
   To use CSS layers when styling Volto, you can define and apply them at the very top level of the page, where they appear in the `<head>` tag.
```
